### PR TITLE
Add console command to read in reservedNames from Yaml file (Fixes: #4)

### PR DESCRIPTION
**Pull request by doobry-systemli**

### CODE REVIEW

File: src/Command/ImportReservedNamesCommand.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            if (null === $repository->findByName($name)) {
+                $output->writeln(
+                    sprintf(
+                        '<INFO>Adding reservedName "%s" to database table</INFO>',
+                        $name
+                    ),
+                    OutputInterface::VERBOSITY_VERBOSE
+                );
+
+                $reservedName = new ReservedName();
```
Comment (0x46616c6b) [inline src/Command/ImportReservedNamesCommand.php]: Here you should create a Factory.

File: src/Command/ImportReservedNamesCommand.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                $reservedName->setName($name);
+
+                $this->manager->persist($reservedName);
```
Comment (0x46616c6b) [inline src/Command/ImportReservedNamesCommand.php]: And a creator.

File: src/DataFixtures/ORM/LoadReservedNameData.php [hunk 3]
```diff
@@ -2,30 +2,60 @@
...
 use Doctrine\Common\Persistence\ObjectManager;
-use Symfony\Component\Yaml\Yaml;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
 
 /**
  * @author doobry <doobry@systemli.org>
  */
-class LoadReservedNameData extends AbstractFixture implements OrderedFixtureInterface
+class LoadReservedNameData extends AbstractFixture implements OrderedFixtureInterface, ContainerAwareInterface
```
[the remark this answers is not in the document]
Comment (doobry-systemli) [reply src/DataFixtures/ORM/LoadReservedNameData.php]: Seems like this is not possible. [`createFixture`](https://github.com/doctrine/data-fixtures/blob/master/lib/Doctrine/Common/DataFixtures/Loader.php#L185-L194) from `Common\DataFixtures\Loader` creates a class object without giving an argument, so I don't see a way to inject something in a Fixture via constructor.

File: src/Command/ImportReservedNamesCommand.php [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+            if (null === $repository->findByName($name)) {
+                $output->writeln(
+                    sprintf(
+                        '<INFO>Adding reserved name "%s" to database table</INFO>',
+                        $name
+                    ),
+                    OutputInterface::VERBOSITY_VERBOSE
+                );
+
+                $reservedNameCreator = new ReservedNameCreator($this->manager, $this->validator, $this->eventDispatcher);
```
[the remark this answers is not in the document]
Comment (doobry-systemli) [reply src/Command/ImportReservedNamesCommand.php]: Indeed, thanks for the hint. Fixed now.

**Pull request merged**